src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props

```diff
@@ -3,5 +3,6 @@
 		<XamarinAndroidVersion>Unknown</XamarinAndroidVersion>
 		<_JavaInteropReferences>Java.Interop;System.Runtime</_JavaInteropReferences>
 		<DependsOnSystemRuntime Condition=" '$(DependsOnSystemRuntime)' == '' ">true</DependsOnSystemRuntime>
+		<MSBuildRuntimeType Condition=" '$(MSBuildRuntimeType)' == '' And '$(OS)' == 'Windows_NT' ">Full</MSBuildRuntimeType>
```

Comment:
Another possibility for the condition here would be `'$(MSBuildAssemblyVersion)' < '15.1'`, since the `MSBuildRuntimeType` built-in property is new to that version.